baseband/gsb/base.py

```diff
@@ -215,6 +215,8 @@ class GSBStreamReader(GSBStreamBase, VLBIStreamReaderBase):
         indexing object is passed, it selects (available) polarizations.  If a
         tuple of objects is passed, the first selects (available) polarizations
         and the second selects channels.  By default, all components are read.
+    fill_value : float or complex
+        Value to use for invalid or missing data (Default value is 0).
```

Comment:
Might as well comment, even though you closed it: write "... data. Default: 0." for consistency with how @cczhu has just been editing lots of other descriptions...

Reply:
But note that for `gsb` I think we should just remove it altogether for now.